src/datamodel/game/player.rs

```diff
@@ -0,0 +1,6 @@
+use crate::datamodel::character::Character;
+
+pub struct Player {
+    pub name: String,
```

Comment:
Why using an `String` instead of the enum `CharacterName`?
Enum types in Rust can be initialiced with an inner value like it's shown in  https://doc.rust-lang.org/book/ch06-01-defining-an-enum.html#enum-values

I suggest to removed name from here, and add to CharactersName this feature.
Issue #9

Reply:
This is not the character's name. It is the player's name. A `Player` represents the person playing the game (eg. Alice or Bob), and a `Character` represents the role that person has in the game (e.g Merlin or Percival). Let me show an example:

```rust
let pato: Player = Player{name: "Pato", character: Box::new(Merlin{})}
```

Reply:
Sorry, i misunderstood this. dismiss this comment